lib/facter/certbot_version.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+Facter.add(:certbot_version) do
+  confine kernel: %w[FreeBSD Linux OpenBSD]
```

Comment:
we should confine on `Facter::Util::Resolution.which('certbot')` and not on the kernel I think?

Reply:
To be honest I copied that over from the other fact, assuming it would limit the fact to the supported operation systems only.